api/src/main/resources/patientPastAppointments.sql

```diff
@@ -1,22 +1,22 @@
 INSERT INTO global_property (property, property_value, description, uuid)
- VALUES ('bahmni.sqlGet.pastAppointments',
-"SELECT
+VALUES ('bahmni.sqlGet.pastAppointments','SELECT
      app_service.name                                                                                AS `DASHBOARD_APPOINTMENTS_SERVICE_KEY`,
      app_service_type.name                                                                           AS `DASHBOARD_APPOINTMENTS_SERVICE_TYPE_KEY`,
-     DATE_FORMAT(start_date_time, \"%d/%m/%Y\")                                                        AS `DASHBOARD_APPOINTMENTS_DATE_KEY`,
-     CONCAT(DATE_FORMAT(start_date_time, \"%l:%i %p\"), \" - \", DATE_FORMAT(end_date_time, \"%l:%i %p\")) AS `DASHBOARD_APPOINTMENTS_SLOT_KEY`,
+     DATE_FORMAT(start_date_time, "%d/%m/%Y")                                                        AS `DASHBOARD_APPOINTMENTS_DATE_KEY`,
+     CONCAT(DATE_FORMAT(start_date_time, "%l:%i %p"), " - ", DATE_FORMAT(end_date_time, "%l:%i %p")) AS `DASHBOARD_APPOINTMENTS_SLOT_KEY`,
      CONCAT(pn.given_name, ' ', pn.family_name)                                                      AS `DASHBOARD_APPOINTMENTS_PROVIDER_KEY`,
      pa.status                                                                                       AS `DASHBOARD_APPOINTMENTS_STATUS_KEY`
 FROM
    patient_appointment pa
    JOIN person p ON p.person_id = pa.patient_id AND pa.voided IS FALSE
    JOIN appointment_service app_service
      ON app_service.appointment_service_id = pa.appointment_service_id AND app_service.voided IS FALSE
+   LEFT JOIN patient_appointment_provider pap on pa.patient_appointment_id = pap.patient_appointment_id AND (pap.voided=0 OR pap.voided IS NULL)
```

Comment:
Hi @binduak 

After taking a look, it seems that the use of "pap" instead of "pa", is intended.
The problem was that the files "patientPastAppointments.sql" and "patientUpcomingAppointments.sql" were both missing a join. 
Keep in mind that this join used to be present in past versions, however was removed a couple PR's ago.

Also another reason that lead me to believe the use of "pap" was intended, was its presence on "patientPastAppointments_v2.sql" and "patientUpcomingAppointments_v2". Files us purpose is to update the queries present in "patientPastAppointments.sql" and "patientUpcomingAppointments.sql"

Reply:
@binduak

Side note, I installed the module using the OpenMrs SDK with MySQL 5.7 and everything seems to work after the fix mentioned above. 
However if you find any issue pls let me know

Reply:
Thanks @icrc-toliveira the reason we don't have join in patientPastAppointments.sql file for pap is, we didn't have patient_appointment_provider table as part of bahmni 0.92 or before.
So we don't require pap in this file at all. 
Will remove the pap part once this PR gets merged.

Reply:
@mks-d @icrc-toliveira @angshu I have tested sql changes with mysql 5.6 and 5.7 this time and everything seems working fine. I will go ahead and merge the PR !